webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Parameters/EditTableRow.js

```diff
@@ -0,0 +1,211 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Tr, Td } from '@patternfly/react-table';
+import { TimesIcon, CheckIcon } from '@patternfly/react-icons';
+import {
+  SelectVariant,
+  Select,
+  SelectOption,
+  Button,
+  TextInput,
+  TextArea,
+  Checkbox,
+  FormGroup,
+  Form,
+  Tooltip,
+} from '@patternfly/react-core';
+import { APIActions } from '../../../../redux/API';
+import { sprintf, translate as __ } from '../../../../common/I18n';
+import { HOST_PARAM, columnNames, typeOptions } from './ParametersConstants';
+import { updateHost } from '../../ActionsBar/actions';
+import { RowActions } from './RowActions';
+
+export const EditParametersTableRow = ({
+  param,
+  rowIndex,
+  setEditingRow,
+  hostId,
+  editHostsPermission,
+  isNew,
+}) => {
+  const [name, setName] = useState(param.name);
+  const [type, setType] = useState(param.parameter_type);
+  const [value, setValue] = useState(param.value);
+  const [selectIsOpen, setSelectIsOpen] = useState(false);
+  const [selectValueIsOpen, setSelectValueIsOpen] = useState(false);
+  const [isHide, setIsHide] = useState(param['hidden_value?']);
+
+  const dispatch = useDispatch();
+
+  const APIActionsProps = {
+    errorToast: ({ response }) =>
+      // eslint-disable-next-line camelcase
+      response?.data?.error?.full_messages || response?.data?.error?.message,
+    handleSuccess: () => {
+      dispatch(updateHost(hostId));
+    },
+  };
+  const onSubmit = () => {
+    if (isNew || param.associated_type !== HOST_PARAM) {
+      dispatch(
+        APIActions.post({
+          url: `/api/hosts/${hostId}/parameters/`,
+          key: 'NEW-PARAM',
+          params: {
+            name,
+            value,
+            parameter_type: type,
+            hidden_value: isHide,
+          },
+          successToast: () =>
+            sprintf(__('Parameter %s has been created successfully'), name),
```

Comment:
Can't it be even shorter? `successfully` feels redundant to me (even though it's used all over the place in Foreman) since you can't `unsuccessfully` create something. I'd even consider
```suggestion
            sprintf(__('Created parameter %s'), name),
```

Reply:
yeah, @MariSvirik always tells us not to use "successfully." I think we've used the pattern `%s created.` elsewhere..

Reply:
Removed the 2 `successfully` in that file, thanks